File: LibraryBooksOfInterest/src/book/of/interest/om/OperationResult.java
package book.of.interest.om;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class OperationResult {
	public enum ErrorEnum
    {
        None,
        InvalidInputData,
        InternalProblem,
        NotAuthenticated, 
        ResultNotInitialized
    }

    public ErrorEnum Error;
    public String ErrorString;

    public OperationResult(ErrorEnum error, String errorString)
    {
        this.Error = error;
        this.ErrorString = errorString;
    }

    public OperationResult()
    {
        this.Error = ErrorEnum.None;
        this.ErrorString = "";
    }
}

File: LibraryExchangeClient/LibraryExchangeClient/BookOfInterest/OperationResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LibraryExchangeClient.BookOfInterest
{
    public class OperationResult
    {
        public enum ErrorEnum
        {
            None,
            InvalidInputData,
            InternalProblem,
            NotAuthenticated,
            ResultNotInitialized
        }

        public ErrorEnum Error;

        public string ErrorString;
    }
}
